com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-basic-unlit-structure.md

```diff
@@ -132,17 +131,46 @@ Tags { "RenderType" = "Opaque" "RenderPipeline" = "UniversalPipeline" }
 
 A SubShader Tag with a name of `RenderPipeline` tells Unity which render pipelines to use this SubShader with, and the value of `UniversalPipeline` indicates that Unity should use this SubShader with URP.
 
+You can create multiple SubShaders with different `RenderPipeline` tags if you wish to execute the same shader in different render pipelines.
+
 For more information on SubShader Tags, see [ShaderLab: SubShader Tags](https://docs.unity3d.com/Manual/SL-SubShaderTags.html).
 
 ### Pass block
 
 In this example, there is one Pass block that contains the HLSL program code. For more information on Pass blocks, see [ShaderLab: Pass](https://docs.unity3d.com/Manual/SL-Pass.html).
 
+A Pass block can optionally contain a Pass Tags block. [ShaderLab: PassTags](https://docs.unity3d.com/Manual/SL-PassTags.html).
+
+When writing passes for URP you need to define the `LightMode` that tells the pipeline which pass to use when executing different parts of the render pipeline.
```

Comment:
If I don't add the LightMode tag, the Pass (and the shader) still works.
What is the default behavior, if I do not set a LightMode tag?